Modules/CodeTrek/Emails/CodetrekMailApplicantRound.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Modules\CodeTrek\Emails;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class CodetrekMailApplicantRound extends Mailable
+{
+    use Queueable, SerializesModels;
+    public $applicationRound;
+    public $codetrekApplicant;
```

Comment:
Can we write like this?
And Please test locally once.
```suggestion
    public $applicationRound, $codetrekApplicant;
```

Reply:
public $applicationRound, $codetrekApplicant;
CI checks will give the error for this .